rabbitmq/values.yaml

```diff
@@ -56,13 +65,13 @@ persistence:
   ## this enables PVC templates that will create one per pod
   enabled: true
 
-  accessMode: ReadWriteOnce
+  #accessMode: ReadWriteOnce
```

Comment:
@dahlo not sure about this change? this prevents multiple RMQ instances using the same PVC?

Reply:
No, a mistake! i have removed the # in my repo now. good catch!